### PR TITLE
feature category and interface

### PR DESCRIPTION
**Pull request by emmakodes**

### CODE REVIEW

File: .gitignore
Comment (frauddetectionsystem) [inline .gitignore:1]: Nice

File: appli/migrations/0005_alter_product_category.py
Comment (frauddetectionsystem) [inline appli/migrations/0005_alter_product_category.py:1]: you changed this? okay

File: appli/static/appli/css/style.css [hunk 1]
```diff
@@ -95,4 +95,36 @@
 .goodnesssec a{
     text-decoration: none;
     text-align: center;
-}
\ No newline at end of file
+}
```
Comment (frauddetectionsystem) [inline appli/static/appli/css/style.css:98]: Okay you added the following

**Review by frauddetectionsystem — APPROVED** [review]
Its okay for now

File: .gitignore
Comment (Trimph-A) [reply .gitignore:1]: Okay sir

**Pull request merged**